packages/event-cache/src/EventCache.js

```diff
@@ -18,18 +18,28 @@ const limiter = new Bottleneck({ maxConcurrent: 25 })
 
 const getPastEvents = memoize(
   async function(instance, fromBlock, toBlock, batchSize = 10000) {
-    if (!instance.loadedCache && instance.ipfsEventCache) {
+    if (
+      instance.ipfsEventCache &&
+      !instance.loadedCache &&
+      (!instance.cacheMaxBlock ||
```

Comment:
Under which scenario would instance.cacheMaxBlock not be set and we would still want to load data from the IPFS cache ? If instance.cacheMaxBlock is not set, doesn't it also mean the config would also not have IPFS hashes to load ?